server.py

```diff
@@ -1,36 +1,46 @@
 """
 Backend server for the report.Ai extension
+
+Make sure that you've added your cluster access in the environment
+mongodb+srv://<username>:<password>@cluster0.dsdb23w.mongodb.net/
 """
 
 # pylint: disable=E0401
+# pylint: disable=W0718
 import os
 import json
 from flask_cors import CORS
-from dotenv import load_dotenv
 from flask import Flask, request, jsonify
 from pymongo import MongoClient
+from pymongo.server_api import ServerApi
 from bson.json_util import dumps
 import requests
 
 
-load_dotenv(".env")
 app = Flask(__name__)
 cors = CORS(app)
 
 
-gpt_neo_key = os.environ.get("GPTNEO")
-mongo_pass = os.environ.get("MONGO")
-
-mongo_uri = "mongodb+srv://mohdansah10:" + mongo_pass + "@cluster0.dsdb23w.mongodb.net/"
+gpt_neo_key = os.environ.get("GPTNEO", None)
+mongo_uri = os.environ.get("MONGO", "mongodb+srv://admin:admin@reportai.ks0reyi.mongodb.net/")
 headers = {"Authorization": gpt_neo_key}

```

Comment:
can removes this line too

Reply:
You want to remove the line defining the URL ?

Reply:
Oh okay, let me try

Reply:
alright works with that line also..